### DISCUSSION TITLE
Widen "Name" Column in Supabase Studio

### DISCUSSION THREAD

**Discussion by jtkeyva** [supabase/supabase#9455] [Feature Requests]
A lot of experts I speak to tend to have longer verbose column names and prefix them with the table name.

ex content table:

content_id
content_creator_account_id
content_updated_at
content_metadata

etc.

would be nice if it was about 1.5- 2x wider so that i can see the whole name at a glance. wouldn't sacrifice the design or aesthetic at such a small increase.

**Comment by Hallidayo** [comment]
Hi @jtkeyva - I've take a look at this and it seems on my studio the columns are quite big by default and extra long column names appear okay.

You can drag the column widths, which then stay the same when you next go into your table editor for the table so they are set and do not revert back to the default, so you could drag out the column widths to what you need and then the next time you go into the table editor you will not need to change the widths as they will be set based on what you set the width to last time.

**Reply by jtkeyva** [reply]
@Hallidayo thanks for checking this out. i should have been more clear...i was referring to slide out pane to update table (that shows all the columns vertically)
